doc/modules/impute.rst

```diff
@@ -52,6 +53,9 @@ The :class:`SimpleImputer` class also supports sparse matrices::
      [6.          3.666...]
      [7.          6.        ]]
 
+Object
+
+
```

Comment:
The paragraph on imputing categorical values stored as string in an array object dtypes is missing :)